docs/oid-info.md

```diff
@@ -189,27 +189,27 @@ that Sigstore operates.
-| server_url + repository + "/actions/runs/" + run_id + "/attempts/" + run_attempt | server_url + project_path + /-/jobs/ + job_id | ??       | ??        | Run Invocation URI           | An immutable identifier that can uniquely identify the build execution     |
-| repository_visibility | project_visibility | ??       | ??        | Source Repository Visibility At Signing | Source repository visibility at the time of signing the certificate |
+| GitHub [(docs)][github-oidc-doc] | GitLab [(docs)](https://docs.gitlab.com/ee/ci/secrets/id_token_authentication.html#token-payload) | CircleCI | Buildkite | Codefresh [(docs)](https://codefresh.io/docs/docs/integrations/oidc-pipelines/) | Fulcio Certificate Extension | Why / Notes / Questions                                                                                                                                                                |
+|--------------------|--------|----------|-----------|-----------|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| aud                | aud     | aud       | aud        | aud        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| iss                | iss     | iss       | iss        | iss        | Issuer                       | This already exists. For example: https://token.actions.githubusercontent.com                                                                                                          |
+| exp                | exp     | exp       | exp        | exp        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| nbf                | nbf     | nbf       | nbf        | nbf        | N/A                          | Only used to validate the JWT. Optional, as per the OIDC spec                                                                                                                         |
+| iat                | iat     | iat       | iat        | iat        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| server_url + job_workflow_ref   | ci_config_ref_uri ([WIP][gitlab-wip-cliams]) | ??       | ??        | platform_url + /build/ +  workflow_id        | Build Signer URI             | Reference to specific build instructions that are responsible for signing. Can be the same as Build Config URI. For example a reusable workflow in GitHub Actions or a Circle CI Orbs. |
```

Comment:
I just want to check that the URLs starting with `platform_url` in this file in this pull request are right - they aren't namespaces by `account_name` or `pipeline_name` or anything like that?

Reply:
Yep, the urls are correct. Accounts in Codefresh are populated upon login (either through the UI or API tokens which are scoped to an account)